scripts/coredump/gdbstubs/arch/risc_v.py

```diff
@@ -50,8 +50,8 @@ class RegNum():
 
 
 class GdbStub_RISC_V(GdbStub):
-    ARCH_DATA_BLK_STRUCT    = "<IIIIIIIIIIIIIIIIII"
-    ARCH_DATA_BLK_STRUCT_2  = "<QQQQQQQQQQQQQQQQQQ"
+    ARCH_DATA_BLK_STRUCT    = "<IIIIIIIIIIIIIIIIIII"
+    ARCH_DATA_BLK_STRUCT_2  = "<QQQQQQQQQQQQQQQQQQQ"
```

Comment:
Add new a struct (or structs) for the changed versions so that the tool will still be able to read older dumps if necessary.

Reply:
> Add new a struct (or structs) for the changed versions so that the tool will still be able to read older dumps if necessary.

Is that a must? Can we state that the coredump should be parsed with the script that came with the version?

Reply:
Can you bump up both of the current "versions" (to 3 and 4)? This will not break reading older dumps and maintain some consistency with other architectures.

Down the line at some point (in some future PR) we can add a new field in the struct to differentiate between 32 and 64 (similar to xtensa)